src/Twig/BackendMenuExtension.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Twig;
+
+use Bolt\Menu\CachedBackendMenuBuilder;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class BackendMenuExtension extends AbstractExtension
+{
+    /** @var CachedBackendMenuBuilder */
+    private $menuBuilder;
+
+    public function __construct(CachedBackendMenuBuilder $menuBuilder)
+    {
+        $this->menuBuilder = $menuBuilder;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('sidebar_menu', [$this, 'getSidebarMenu']),
+        ];
+    }
+
+    public function getSidebarMenu($pretty = false): string
+    {
+        return $this->menuBuilder->getMenu($pretty);
```

Comment:
here should go json encoding

Reply:
Sure? json _decoding_ is fast.. json _encoding_ is slow.. I've done the encoding inside the part that's cached for performance.

Reply:
is it really that slow to encode a simple array?

Reply:
Well, "slow" is relative of course.. After comtemplating it, i suppose it'd be a "micro optimisation" if we cache it after encoding, but in practice i couldn't measure the difference in performance. I'll modify it.

Reply:
I think it was slow before PHP 7

Reply:
done